Policies/Kubernetes/container-allowed-capabilities/rego/src_test.rego

```diff
@@ -0,0 +1,263 @@
+    input := { "review": input_review([cadd(["one"])]), "parameters": {"allowedCapabilities": []}}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_none_allowed_undefined {
+    input := { "review": input_review([cadd(["one"])]), "parameters": {}}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_none_allowed_undefined_x2_x2 {
+    input := { "review": input_review([cadd(["one", "two"]), cadd(["three", "two"])]), "parameters": {}}
+    results := violation with input as input
+    trace(sprintf("results are: %v", [results]))
+    count(results) == 2
+}
+test_input_disallowed_x1 {
+    input := { "review": input_review([cadd(["three"])]), "parameters": {"allowedCapabilities": ["one", "two"]}}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_disallowed_x2_just_one {
+    input := { "review": input_review([cadd(["one"]), cadd(["three", "two"])]), "parameters": {"allowedCapabilities": ["one", "two"]}}
+    results := violation with input as input
+    count(results) == 1
+}
+test_input_disallowed_x2 {
+    input := { "review": input_review([cadd(["three"]), cadd(["three", "two"])]), "parameters": {"allowedCapabilities": ["one", "two"]}}
+    results := violation with input as input
+    count(results) == 2
+}
+
+
+test_input_empty_drop {
+   input := { "review": input_review([cdrop(["one", "two"])]), "parameters": {"requiredDropCapabilities": []}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_all_dropped {
+   input := { "review": input_review([cdrop(["one", "two"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_extra_dropped {
+   input := { "review": input_review([cdrop(["one", "two", "three"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_all_dropped_x2 {
+   input := { "review": input_review([cdrop(["one", "two"]), cdrop(["one", "two"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 0
+}
+test_input_missing_drop {
+   input := { "review": input_review([cdrop(["two"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 1
+}
+test_input_one_missing_drop_x2 {
+   input := { "review": input_review([cdrop(["one"]), cdrop(["one", "two"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 1
+}
+test_input_missing_drop_x2 {
+   input := { "review": input_review([cdrop(["one"]), cdrop(["two"])]), "parameters": {"requiredDropCapabilities": ["one", "two"]}}
+   results := violation with input as input
+   count(results) == 2
+}
+test_input_drop_undefined_x2 {
```

Comment:
Can you please add a test for input drop undefined and requiredDropCapabilities is empty array?